src/c++/perf_analyzer/infer_context.h

```diff
@@ -185,17 +161,6 @@ class InferContext {
   /// \param delayed Whether the request fell behind its scheduled time.
   void SendRequest(const uint64_t request_id, const bool delayed);
 
-  /// Helper function to prepare the InferContext for sending
-  /// inference request.
-  /// \param ctx The target InferContext object.
-  /// \return cb::Error object indicating success or failure.
-  cb::Error PrepareInfer(InferContext* ctx);
-
-  /// Helper function to prepare the InferContext for sending
-  /// inference request in shared memory. \param ctx The target
-  /// InferContext object. \return cb::Error object indicating
-  /// success or failure.
-  cb::Error PrepareSharedMemoryInfer(InferContext* ctx);
```

Comment:
Shouldn't CreateInferInput (a few lines down) be removed as well?

Reply:
removed